wrapping/python/openmeeg/openmeeg.i

```diff
@@ -38,6 +38,12 @@
 %include <windows.i>
 #endif
 
+#ifdef _MSC_VER
+#pragma warning( push )
+#pragma warning( disable:4127 )  /* conditional expression is constant */
+#pragma warning( disable:4701 )  /* potentially uninitialized local variable 'v' used */
```

Comment:
And then a surprising few number of extra ignores for the code that SWIG generates